docs/management/cases/setup-cases.asciidoc

```diff
@@ -18,6 +18,12 @@ a|
 NOTE: The *Actions and Connectors* feature privilege is required to create, add,
 delete, and modify case connectors and to send updates to external systems.
 
+| Give assignee access to cases
```

Comment:
No part of this PR but maybe we can add an action when a user has full access to cases except the ability to delete cases and comments.

Reply:
Yup!  I'll open a subsequent PR